lib/EasyDirectiveVisitor.ts

```diff
@@ -359,8 +359,17 @@ abstract class EasyDirectiveVisitor<
     query: GraphQLObjectType<any, TContext>,
     schema: GraphQLSchema,
     directiveName: string,
-  ): TLocation extends DirectiveLocation.QUERY ? void : never {
-    throw new Error('Method not implemented.');
+  ): GraphQLObjectType<any, TContext> {
+    return query;
+  }
+
+  // istanbul ignore next (should be overridden and never reached)
+  public visitMutation(
+    mutation: GraphQLObjectType<any, TContext>,
+    schema: GraphQLSchema,
+    directiveName: string,
+  ): GraphQLObjectType<any, TContext> {
+    return mutation;
```

Comment:
nit: wouldn't it be better to let these functions throw an error if not overwritten by the directives that can be used on `queries` and `mutations` ?